### PR TITLE
commits fetch error fixed

### PR DESCRIPTION
**Pull request by pain11**

### CODE REVIEW

File: app/models/commits_fetcher.rb [hunk 1]
```diff
@@ -9,7 +9,7 @@ def initialize(repo, user, round)
   end
 
   def fetch(type = :daily)
-    user.gh_client.repos.branches(repo.owner, repo.name, {auto_pagination: true}).each do |branch|
+    user.gh_client.repos.branches(user: user.github_handle, repo: repo.name).list.each do |branch|
```
Comment (pratik14) [inline app/models/commits_fetcher.rb]: Why did you change repo.owner to user.github_handle?

Comment (pain11) [reply app/models/commits_fetcher.rb]: It is easy to track.

**Comment by pratik14** [comment]
Note: Test are failing because of Git Api rate limit.
